modules/apps/export-import/export-import-api/src/main/java/com/liferay/exportimport/vulcan/batch/engine/ExportImportVulcanBatchEngineTaskItemDelegate.java

```diff
@@ -19,6 +22,8 @@ public default List<String> getNestedFields() {
 		return null;
 	}
 
+	public Map<String, Serializable> getParameters();
```

Comment:
I think here is better to have
```
public default Map<String, Serializable> getParameters() {
		return Collections.emptyMap();
	}
```
so we don't have to override in all the points just to return the empty map